extensions/renderer/resources/app_window_custom_bindings.js

```diff
@@ -30,9 +30,10 @@ var try_hidden = function (view) {
 };
 
 var try_nw = function (view) {
-  if (view.nw)
-    return view;
-  return privates(view);
+  if (view.nw) {
+    return true;
+  }
+  return false;
 };
```

Comment:
I see, but I was wondering the reason of other code call try_nw() is that they want to verify if the "view" has nw.window? 
In this case, how this code would break it? the original code only return the same "view".